spark/src/main/java/io/ddf/spark/DelegatingDDFManager.java

```diff
@@ -0,0 +1,63 @@
+package io.ddf.spark;
```

Comment:
No, it's used in PE for CreateDDF API as well, as the design doc specify that there a one and only one mapping between data source <==> DDFManager, thus a singleton SparkDDFManager cannot be used for different source directly. Hence the need for this DelegatingDDFManager class.